tcms/core/history.py

```diff
@@ -17,17 +17,15 @@ def diff_objects(old_instance, new_instance, fields):
     full_diff = []
 
     for field in fields:
-        field_diff = []
         old_value = getattr(old_instance, field.attname)
         new_value = getattr(new_instance, field.attname)
-        for line in difflib.unified_diff(
-            str(old_value).split("\n"),
+        field_diff = [line for line in difflib.unified_diff(
+        str(old_value).split("\n"),
             str(new_value).split("\n"),
             fromfile=field.attname,
             tofile=field.attname,
             lineterm="",
-        ):
-            field_diff.append(line)
+        )]
         full_diff.extend(field_diff)
```

Comment:
Function diff_objects refactored with the following changes:

-  Convert for loop into list comprehension ([list-comprehension](https://docs.sourcery.ai/refactorings/list-comprehension/))

Reply:
The indent on the new line 23 looks wrong. I expect it to be indented by 4 more spaces